openflow_input/nicira_exit

```diff
@@ -0,0 +1,16 @@
+#version any
```

Comment:
@rsharo sorry for the delay on this pull request. Do these apply to all OpenFlow versions? Everything looks good to me except for this (in each input file).